src/app/modals/add-to-group/add-to-group.component.ts

```diff
@@ -245,7 +245,7 @@ export class AddToGroupModal implements OnInit, OnDestroy {
   public searchGroups(event?): void {
     // Execute search after every third character of the search term
     // Use ">=" instead of ">" so that when we have empty search term (when we type something and delete it), we will reload the groups
-    if ((this.groupSearchTerm.length >= 0) && (this.groupSearchTerm.length % 3 === 0)) {
```

Comment:
@rehanabbasi @codiak Isn't this so we're not searching on every keypress? Can we think of a different way here?

Reply:
Kelsey told me yesterday that she wants to search on every keypress. She added this to the ticket.